src/styles/theme.scss

```
@@ -48,12 +42,13 @@ $zTopBar: 35;
 $zDropdown: 40;
 
 /** Component dimensions **/
-$headerHeight: 54px;
+$headerHeight: 36px;
 $mapUtilButtonHeight: 44px;
 $loadingStateDefaultHeight: 200px;
 
 $chartCardMaxWidth: 500px;
```

Stylelint (17.13.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)